synfig-studio/plugins/lottie-exporter/layers/text.py

```diff
@@ -0,0 +1,141 @@
+"""
+Will store all the functions corresponding to text layer in lottie
+"""
+
+import sys
+import settings
+import copy
+from lxml import etree
+from common.Count import Count
+from synfig.group import get_additional_width,get_additional_height
+from helpers.transform import gen_helpers_transform
+from helpers.blendMode import get_blend
+sys.path.append("..")
+
+def calc_default_text_properties(layer):
+    """
+    Generated the lottie dictionary corresponding to properties which won't change in text-animation
+
+    Args:
+        layer  (common.Layer.Layer) : Synfig format text layer
+
+    Returns:
+        Lottie format default dictionary for text animation properties
+    """
+    default_dict = {
+      "nm": "",
+      "s": {
+        "t": 0,
+        "xe": {
```

Comment:
See if you can use this function here: https://github.com/synfig/synfig/blob/334b5b9b6d888ad68bf6b1f3202f581cb3a3e14d/synfig-studio/plugins/lottie-exporter/properties/value.py#L11
This might reduce the code lines. :D